Contributing.md

```diff
@@ -131,38 +130,38 @@ When you're finished with your work and are ready to have it merged into the mai
+For example, if your local branch X was successfully merged into the OfficeDev/office-js-docs-pr main branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/office-js-docs-pr main branch. To do this, open Git Bash and execute the following commands.
 
-    cd microsoft-graph-docs
-    git pull upstream master:X2
+    cd office-js-docs-pr
+    git pull upstream main:X2
     git push origin X2
 
 You now have local copies (in a new local branch) of the work that you submitted in branch X. The X2 branch also contains all the work other writers have merged, so if your work depends on others' work (for example, shared images), it is available in the new branch. You can verify that your previous work (and others' work) is in the branch by checking out the new branch...
 
     git checkout X2
 
-...and verifying the content. (The `checkout` command updates the files in `C:\Users\<yourusername>\microsoft-graph-docs` to the current state of the X2 branch.) Once you check out the new branch, you can make updates to the content and commit them as usual. However, to avoid working in the merged branch (X) by mistake, it's best to delete it (see the following **Delete a branch** section).
+...and verifying the content. (The `checkout` command updates the files in `C:\Users\<your username>\office-js-docs-pr` to the current state of the X2 branch.) Once you check out the new branch, you can make updates to the content and commit them as usual. However, to avoid working in the merged branch (X) by mistake, it's best to delete it (see the following **Delete a branch** section).
 
 #### Delete a branch
 
 Once your changes are successfully merged into the main repository, delete the branch you used because you no longer need it.  Any additional work should be done in a new branch.  
 
 #### To delete a branch
 
-1. In the Git Bash command prompt, type `git checkout master`. This ensures that you aren't in the branch to be deleted (which isn't allowed).
+1. In the Git Bash command prompt, type `git checkout main`. This ensures that you aren't in the branch to be deleted (which isn't allowed).
 2. Next, at the command prompt, type `git branch -d <branch name>`. This deletes the branch on your computer only if it has been successfully merged to the upstream repository. (You can override this behavior with the `–D` flag, but first be sure you want to do this.)
-3. Finally, type `git push origin :<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your github fork.  
+3. Finally, type `git push origin:<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your GitHub fork.  
```

Comment:
IIRC, we need to keep this space

```suggestion
3. Finally, type `git push origin :<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your GitHub fork.  
```